modules/op-release-notes-1-16.adoc

```diff
@@ -0,0 +1,594 @@
+
+With this update, {pipelines-title} General Availability (GA) 1.16 is available on {OCP} 4.15 and later versions.
+
+[id="new-features-1-16_{context}"]
+== New features
+
+In addition to fixes and stability improvements, the following sections highlight what is new in {pipelines-title} 1.16:
+
+[id="pipelines-new-features-1-16_{context}"]
+=== Pipelines
+
+* With this update, you can configure the resync period for the pipelines controller. Once every resync period, the controller reconciles all pipeline runs and task runs, regardless of events. The default resync period is 10 hours. If you have a large number of pipeline runs and task runs, a full reconciliation every 10 hours might consume too many resources. In this case, you can configure a longer resync period.
++
+.Example of configuring a resync period of 24 hours
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  pipeline:
+    options:
+      deployments:
+        tekton-pipelines-controller:
+          spec:
+            template:
+              spec:
+                containers:
+                - name: tekton-pipelines-controller
+                  args:
+                    - "-resync-period=24h" 
+----
+
+* With this update, when defining a pipeline, you can set the `onError` parameter for a task to `continue`. If you make this setting and the task fails when the pipeline is executed, the pipeline logs the error and continues to the next task. By default, a pipeline fails if a task in it fails. 
++
+.Example of setting the `onError` parameter. After the `task-that-fails` task fails, the `next-task` executes
+[source,yaml]
+----
+apiVersion: tekton.dev/v1 
+kind: Pipeline 
+metadata:
+  name: example-onerror-pipeline
+spec:
+  tasks:
+  - name: task-that-fails
+    onError: continue
+    taskSpec:
+      steps:
+      - image: alpine
+        name: exit-with-1
+        script: |
+          exit 1
+  - name: next-task
+# ...
+----
+
+* With this update, if a task fails, a `finally` task can access the `reason` parameter in addition to the `status` parameter to distinguish if the failure was allowed or not. You can access the `reason` parameter  through `$(tasks.<task_name>.reason)`. If the failure is allowed, the `reason` is set to`FailureIgnored`, if the failure is not allowed, the `reason` is set to `Failed`. This additional information can be used to identify that the checks failed, but the failure can be ignored. 
+
+* With this update, larger results are supported through sidecar logs as an alternative to the default configuration, which limits results to the size of 4 KB per step and 12 KB per task run. To enable larger results using sidecar logs, set the `results-from` feature flag to `sidecar-logs` by running the following command:
++
+[source,terminal]
+----
+$ oc patch cm feature-flags -n openshift-pipelines -p '{"data":{"results-from":"sidecar-logs"}}' 
```

Comment:
This was approved by @vdemeester but I'd appreciate if Vincent took another look - I thought patching a configmap directly might not work on the product install?